modules/govuk/manifests/apps/search_api.pp

```diff
@@ -306,4 +277,11 @@
       varname => 'TENSORFLOW_SAGEMAKER_VARIANTS',
       value   => $tensorflow_sagemaker_variants;
   }
+
+  if $elasticsearch_b_uri != 'null' {
```

Comment:
Out-of-scope: it looks like this [was removed](https://github.com/alphagov/search-api/commit/aef1da207bc6183e1681c405b8883f29a2d6fe56). The variable is no longer present in production, so we could just remove it here.

Reply:
Oh great - nice catch.

Reply:
I've pushed a change up for this, if you could take another look